server/server/src/cold_chain/mod.rs

```diff
@@ -0,0 +1,77 @@
+use actix_web::{
+    web::{self},
+    HttpRequest, Result,
+};
+use service::{
+    auth::{validate_auth, AuthDeniedKind, AuthError, Resource, ResourceAccessRequest},
+    auth_data::AuthData,
+    service_provider::{ServiceContext, ServiceProvider},
+    user_account::UserAccountService,
+};
+
+mod login;
+mod sensor;
+use login::login;
+use sensor::sensors;
+
+const URL_PATH: &str = "/coldchain/v1";
+const COOKIE_NAME: &str = "coldchain";
+
+pub fn config_cold_chain(cfg: &mut web::ServiceConfig) {
+    cfg.route(&format!("{}/login", URL_PATH), web::post().to(login));
+    cfg.route(&format!("{}/sensor", URL_PATH), web::put().to(sensors));
+}
```

Comment:
Just a note, if we have more REST like endpoints like this, there are crates similar to the graphq crate we are using to specify the API a bit better. 

For example: https://github.com/juhaku/utoipa/blob/15053c53b49d5990c230a80517fba642a7cfcaa5/examples/todo-actix/src/todo.rs
This would allow us to generate an OpenAPI spec file from our Rust code which can then be used to provide nice auto generated API docs or auto generated JS client libs.

The linked lib looks actually fairly simple to use in case we want to do it now...